src/style.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 700;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-weight: 300;
+  font-family: Avenir, sans-serif;
+}
+
+/* Big search styles */
+.search-panel__big {
+  margin-bottom: 20px;
+  margin-top: 20px;
+}
+
+.search-bar__big {
+  width: 100%;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 4px;
+  padding-left: 62px;
+  background: url("./images/Search.svg") 26px 19px no-repeat;
+  background-position: 26px center;
+  transition: all 0.5s;
+}
+
+.search-bar__big::placeholder {
+  font-family: Avenir, serif;
+  font-size: 16px;
+  color: #3c4c5f;
+}
+
+.search-bar__big:hover {
+  box-shadow: 0 3px 8px 0 #3d4e6133;
+}
+
+.search-bar__big:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  font-weight: 700;
+  background-image: url("./images/Search.svg");
```

Comment:
you already added this style on search-bar__big class

Reply:
when I remove these styles from  : focus the search icon disappears